playwright/tests/search/test_search.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from playwright.sync_api import Page, expect
```

Comment:
If you can add a README.md in the playright folder to tell other how to invoke the run locally will be great, thanks